tests/test_job_manager.py

```diff
@@ -0,0 +1,50 @@
+from unittest.mock import patch
+import pytest
+from datetime import datetime
+from metriq_gym.job_manager import JobManager, MetriqGymJob
+from tests.test_schema_validator import TEST_BENCHMARK_NAME, TestJobType
+
+
+@pytest.fixture(autouse=True)
```

Comment:
[question]: Do we want to put these fixtures in a `conftest`? It may be premature as I don't think (at the moment) that we have any other tests that could use these fixtures, so perhaps better to keep them local for now.

Reply:
this patch is specific to the job_manager, so I would keep it here for now